packages/block-library/src/heading/variations.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * WordPress dependencies
+ */
+import { sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import HeadingLevelIcon from './heading-level-icon';
+
+const variations = [ 1, 2, 3, 4, 5, 6 ].map( ( level ) => ( {
+	name: `heading-${ level }`,
+	/* translators: %d: heading level. */
+	title: sprintf( 'Heading %d', level ),
```

Comment:
It isn't translated with `_` or `_x` (if we want to have the context).

Reply:
I don't understand your comment 😄 . You mean I should use `_x`? I used `sprintf` because of the param in the translatable string.

Reply:
`sprintf` is only a helper that helps to inject data into translations, you need to combine it with a function that translates the string 😄

Reply:
Oh, wow - thanks for catching this Grzegorz!